### ISSUE TITLE
Question about freeze_bn

### ISSUE THREAD

**Issue by nijkah** [dbolya/yolact#210]
https://github.com/dbolya/yolact/blob/master/yolact.py#L578
Line 578~582

To my knowledge, when the model uses pre-trained backbone, it is common to freeze batchnorm in backbone.

But above code, it seems to freeze all batchnorms of the yolact when training mode.

Am I missing or confused with something??

**Comment by dbolya** [comment]
That's cause there aren't any other batch norms than in the backbone, so freezing all batch norms in yolact would freeze just the batch norms in the backbone.

Note though, that by default if the batch size is > 6 per GPU, batch norm by default is not frozen so this doesn't get called. We can do that since our default batch size is 8, which is much larger than other instance segmentation methods.

**Comment by nijkah** [comment]
I didn't know the later thing. Thank you for sharing that.

But, i still wonder, it seems that there is one batchnorm in PredictionModule.
Is it exception or not trained?

**Comment by dbolya** [comment]
That branch is never used
